### PR TITLE
Implemented the top section of Impacts page

### PR DESCRIPTION
**Pull request by kerry918**
* Created card components for each part (BannerCard)
* Used grid to display the cards for the banner (Banner)
* The numbers for those cards might need an update

### CODE REVIEW

**Comment by kerry918** [comment]
Close #95

**Comment by jack-lin-jql** [comment]
> #95

you mean close #92? lol

**Comment by jack-lin-jql** [comment]
There a slight misalignment for the top and the bottom between the left and right side of the entire section
image

File: src/components/ImpactBanner/Banner.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import BannerCard from "./BannerCard";
+import useStyles from "./Banner-jss";
+import Events from "../../assets/images/impact/events.png";
```
Comment (jack-lin-jql) [inline src/components/ImpactBanner/Banner.js:4]: nit: would rename them to something indicative of an image, perhaps `eventsImg`. Note the non-capitalization of the first char as well.
This is so we don't confuse ourselves with this being a React component or a class

File: src/components/ImpactBanner/Banner.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+import Event_Attend from "../../assets/images/impact/event_attend.png";
```
Comment (jack-lin-jql) [inline src/components/ImpactBanner/Banner.js:5]: I think the standard across this repo is to use camelCase, so if we can keep the consistency, that would be great

File: src/components/ImpactBanner/Banner.js [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+import Community from "../../assets/images/impact/community.png";
+import First_Year from "../../assets/images/impact/first_year.png";
+import Mentorship from "../../assets/images/impact/mentorship.png";
+import Coffee from "../../assets/images/impact/coffee.png";
+
+export default function Banner() {
+  const classes = useStyles();
+  const cards = [
+    {
+      container: classes.eventContainer,
```
Comment (jack-lin-jql) [inline src/components/ImpactBanner/Banner.js:15]: nit: can we rename this to containerClassname to ensure no ambiguity

File: src/components/ImpactBanner/Banner.js [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+      number: "15",
+      desc: "Events",
+      img: Events,
+      boxStyle: { width: "55%" },
+      smallText: { fontWeight: "600", fontSize: "2.2vw" },
```
Comment (jack-lin-jql) [inline src/components/ImpactBanner/Banner.js:20]: nit: can we also move the classes into the jss file 🙏

File: src/components/ImpactBanner/Banner.js [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+import First_Year from "../../assets/images/impact/first_year.png";
+import Mentorship from "../../assets/images/impact/mentorship.png";
+import Coffee from "../../assets/images/impact/coffee.png";
+
+export default function Banner() {
+  const classes = useStyles();
+  const cards = [
+    {
+      container: classes.eventContainer,
+      number: "15",
```
Comment (jack-lin-jql) [inline src/components/ImpactBanner/Banner.js:16]: I would rename this to `stat` or `count` (prob count is better). A few reasons:
1. `number` is also a primitive type in JS, so we should avoid naming anything directly as "number" to avoid name collision/type vs variable confusion
2. the possible value types of this property isn't just number, as you are passing them in as strings

File: src/components/ImpactBanner/Banner.js (hunk 4 above)
Comment (jack-lin-jql) [inline src/components/ImpactBanner/Banner.js:20]: Also, are you using `smallText` anywhere?

File: src/components/landing_page/Content.js [hunk 6]
```diff
@@ -88,7 +88,7 @@ export default function Content() {
         </Grid>
       </Grid>
 
-      <Grid container className={classes.content}>
+      <Grid container spacing={3} className={classes.content}>
```
Comment (jack-lin-jql) [inline src/components/landing_page/Content.js:91]: why was this modified?

File: src/components/impact/Impact.js [hunk 7]
```diff
@@ -1,10 +1,12 @@
 import React from "react";
 import GrowthStory from "./GrowthStory";
 import Testimonials from "./Testimonials";
+import Banner from "../ImpactBanner";
 
 export default function Impact() {
   return (
     <div>
+      <Banner />
```
Comment (jack-lin-jql) [inline src/components/impact/Impact.js:9]: very nice on breaking the banner out of the Testimonial component!

**Comment by jack-lin-jql** [comment]
great work, please address my comments accordingly, thanks!

File: src/components/ImpactBanner/Banner-jss.js [hunk 8]
```diff
@@ -38,6 +39,54 @@ const useStyles = makeStyles((theme) => ({
     gridColumnEnd: "span 6",
     gridRowEnd: "span 3",
   },
+  eventBoxStyle: {
+    background: "#020B2C",
+    color: "#FFFFFF",
+    display: "flex",
+    flexDirection: "column",
+    justifyContent: "flex-end",
+    width: "55%",
```
Comment (jack-lin-jql) [inline src/components/ImpactBanner/Banner-jss.js:48]: see how this class here comparing with the rest of the boxStyles only differ in width. We can create a generic class perhaps called `boxStyle` and then have specific widths for each type of box. 

E.g.
```
boxStyle: {
    background: "#020B2C",
    color: "#FFFFFF",
    display: "flex",
    flexDirection: "column",
    justifyContent: "flex-end",
},
eventCardWidth: {
    width: "55%"
},
firstYearCardWidth: {
    width: "35%"
},
...
```

In the component, we can then concat classes together. A common library used to do this concatenation is https://github.com/JedWatson/classnames which saves us from doing manual class string generation such as `[classes.style1, classes.style2].join(' ')`. It additionally allows easy conditionals.

Let me know if this doesn't make sense.

**Comment by jack-lin-jql** [comment]
Sorry I should have made this more clear, but the misalignment issue also presents and the bottom of the banner. 
image

**Comment by jack-lin-jql** [comment]
2 more comments 😄  otherwise you're good 💯

**Review by jack-lin-jql — APPROVED** [review]
LGTM! 👍
